bokeh/server/application_context.py

```diff
@@ -18,7 +18,7 @@
 class BokehServerContext(ServerContext):
     def __init__(self, application_context):
         self.application_context = application_context
-        self._callbacks = _CallbackGroup() #self.application_context.io_loop)
+        self._callbacks = _CallbackGroup(self.application_context.io_loop)
```

Comment:
I changed this back to make this issue work. Thus, I'm curious, @quasiben what you were trying to do here?